drivers/metadata/postgres/metadata.go

```diff
@@ -277,11 +277,24 @@ SELECT
   c.relname as "Name",
   CASE i.indisprimary WHEN TRUE THEN 'YES' ELSE 'NO' END,
   CASE i.indisunique WHEN TRUE THEN 'YES' ELSE 'NO' END,
-  CASE c.relkind WHEN 'r' THEN 'table' WHEN 'v' THEN 'view' WHEN 'm' THEN 'materialized view' WHEN 'i' THEN 'index' WHEN 'S' THEN 'sequence' WHEN 's' THEN 'special' WHEN 'f' THEN 'foreign table' WHEN 'p' THEN 'partitioned table' WHEN 'I' THEN 'partitioned index' END as "Type"
+  COALESCE(am.amname, 
+  	CASE c.relkind 
+		WHEN 'r' THEN 'table' 
```

Comment:
Nit: there's a condition below to only include `relkind` of `i`, `I` and empty. Can you remove other cases here?